ICSharpCode.Decompiler/IL/Transforms/DelegateConstruction.cs

```diff
@@ -239,7 +239,40 @@ protected override void Default(ILInstruction inst)
 					child.AcceptVisitor(this);
 				}
 			}
-			
+
+			// Handle variables in ExpressionTrees
+			// call Field(call Constant(ldloc V_0, call GetTypeFromHandle(ldtypetoken ExpressionTrees+<>c__DisplayClass56_0)), call GetFieldFromHandle(ldmembertoken i)) at IL_0114
+			// -> call Constant(ldloc i, call GetTypeFromHandle(ldtypetoken System.Int32))
+			protected internal override void VisitCall(Call inst)
+			{
+				if (inst.Method.FullName.Equals("System.Linq.Expressions.Expression.Field") &&
```

Comment:
Putting the logical and operator at the beginning of the line would make it easier to read the code.